code/__DEFINES/combat.dm

```diff
@@ -10,6 +10,18 @@
 #define STAMINA 	"stamina"
 #define BRAIN		"brain"
 
+//damage flags
+#define MELEE 		"melee"
+#define BULLET 		"bullet"
+#define LASER 		"laser"
+#define ENERGY 		"energy"
+#define BOMB 		"bomb"
+#define BIO 		"bio"
+#define RAD 		"rad"
+#define FIRE 		"fire"
+#define ACID 		"acid"
+#define MAGIC		"magic"
```

Comment:
I understand the sentiment, but this is the way /tg/ does this and I feel like this would end up being really confusing to someone who had to port code with these in the future. Do you think it's worth the sacrifice here?

Reply:
If this was being done from scratch then I'd definitely say yes. Since it's from TG though the point about it adding confusion when porting items is a valid one.
I consider it worth the trade off personally. There's already a lot of things that need to be changed when porting something from another server, and with how widespread the armour system is in the code (as evidenced by this PR) it'd be easy to find an example of what it should be changed to.